src/types/fractional_scale_v1.zig

```diff
@@ -0,0 +1,22 @@
+const wlr = @import("../wlroots.zig");
+
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+
+pub const FractionalScaleManagerV1 = extern struct {
+    global: *wl.Global,
+
+    events: extern struct {
+        destroy: wl.Signal(void),
```

Comment:
```suggestion
        destroy: wl.Signal(*FractionalScaleManagerV1),
```
I think that should be that here

Reply:
This is a copy-paste from `viewporter.zig`, is it wrong too?

Reply:
Copy pasting is always wrong, the wlroots source code must be examined to determine what pointer if any is passed as the data argument when signals are emitted.

Reply:
Okay, wlroots passes NULL. void is good then?

Reply:
Yep, void is correct